### ISSUE TITLE
[IMPORTANTE] Até mais

### ISSUE THREAD

**Issue by haskellcamargo** [haskellcamargo/gemidao-do-zap#37]
## E obrigado pelos peixes

Ao que parece, a TotalVoice restringiu a API de executar ligações para qualquer número que não seja o seu cadastrado, o que inutiliza totalmente o _Gemidão do Zap™_.

Foi um prazer servir com vocês.
![2017-08-10-115752_1025x45_scrot](https://user-images.githubusercontent.com/7553006/29176759-27880b5c-7dc3-11e7-9dd4-24d386b28cbc.png)

**Comment by williamokano** [comment]
Opções? Twillio e DirectCall? Ou it's dead for good?

**Comment by darlanxw** [comment]
Na verdade, eles colocaram uma validação a mais de cadastro, agora voce precisa fazer uma confirmação por email, confirmação por telefone fixo e mais algumas coisas, ai ele libera pra utilizar o crédito e fazer as chamadas

**Comment by murilo-campaner** [comment]
Então, mas estou à 10 minutos tentando receber o e-mail de confirmação (já confirmei telefone fixo e celular) e até agora não recebi o e-mail (já testei em 3 emails diferentes)

**Comment by darlanxw** [comment]
Aqui chegou bem rápido... Fiz o teste novamente e fez a ligação normalmente depois de confirmar tudo que era necessário

**Comment by murilo-campaner** [comment]
Beleza, agora deu certo, confirmei a conta, mas quando executo o comando no terminal aparece "Erro: conta desativada"

**Comment by castroalves** [comment]
O email de confirmação está vindo com a URL errada, caindo num erro 404. É só trocar no início `api.totalvoice.com.br` para `api2.totalvoice.com.br`.

Ontem eu criei um app parecido com a API do Twilio. Eles dão **US$ 20,00** de créditos.

Só que tem dois inconvenientes: 1) Você precisa cadastrar o número do destinatário e pedir pra ele te mandar um código de confirmação que ele recebe via SMS; e 2) Na ligação, rola uma mensagem em inglês antes e dps o usuário tem que digitar qualquer tecla pra ouvir a mensagem (no caso, o Gemidão).

Pra mim valeu como aprendizado. Estou dando meus primeiros passos com NodeJS e, de quebra, tive a oportunidade de aprender um pouco da API de voz do Twilio. 

Por sinal, ela é bem simples. Meu app tem apenas 10 linhas.

**Comment by murilo-campaner** [comment]
Ah, agora perdeu a graça

**Comment by jhonata-menezes** [comment]
Quando eu fiz o cadastro ativei por email e enviei um email tirando dúvidas. Eles ligaram e desbloquearam a conta para qualquer ligação.
Eles informaram que esta decisão foi tomada por que tiveram problemas com pessoas fazendo o cadastro  e usando para fazer o "mal"

**Comment by murilo-campaner** [comment]
Mandei um e-mail dizendo que gostaria até de colocar créditos para entrar na brincadeira. Vamos ver o que respondem haha

**Comment by haskellcamargo** [comment]
@williamokano adiciona ao README essa informação? Falando que tem mais passos para validação, que nem sempre estão chegando os R$ 3,00 e que a brincadeira "acabou"? :cry:

**Comment by williamokano** [comment]
Sim, adiciono assim que chegar em algum PC :(

Em 10 de ago de 2017 12:38 PM, "Marcelo Camargo" <notifications@github.com>
escreveu:

**Comment by takita-jonathan** [comment]
okay, mas oque parece a variavel deles nao limpa o seu ultimo SMS token que recebeu, entao se colocar a confimaçao pelo fixo, o mesmo token que recebeu para confirmar o celular, funciona, pelo menos funcionou no meu.

**Comment by haskellcamargo** [comment]
Aqui voltou a funcionar do nada. A mensagem de validação sumiu.

**Comment by darlanxw** [comment]
Verdade, ao se cadastrar não tem mais a validação extra, aparentemente ta ok...

**Comment by williamokano** [comment]
Botei uns créditos aqui e está funfando de boa ainda.

Será que é só nos R$ 3,00 free?

**Comment by ezosk** [comment]
Aqui o email de ativação não chega nem fudendo. Alguém que conseguiu essa ativação extra, após isto, volta a funcionar?

**Comment by williamokano** [comment]
Conseguiu, tente entre 08 e 17 pra não cair no antispam

Em 10 de ago de 2017 9:27 PM, "ezosk" <notifications@github.com> escreveu:

**Comment by akamuraasai** [comment]
Criei uma conta hoje, recebi os R$3,00 na hora. Mas ele pediu uma validação de SMS para funcionar. Fiz a validação em dois minutos e ele funcionou tranquilamente. Fiz umas 10 ligações e quando eu só tinha cerca de R$0,60 de credito ele começou a dar um erro de que eu só poderia enviar ligações para o meu número, ou qualquer coisa assim. Entrei no painel e dessa vez ele me pediu uma confirmação via e-mail e alguns dados pessoais (CPF e endereço).
Completei esses passos (o e-mail chegou em 2 minutos) e a API foi liberada novamente e eu consegui acabar com o meu saldo remanescente.
Todos esses testes foram feitos aproximadamente entre 18:15 e 18:45.

O serviço da API deles deve apenas estar instável pelo grande tráfego, todos os problemas que surgiram devem ter relação com isso.

**Comment by Deivide** [comment]
Agora estão pedindo documentos pessoais para confirmar o cadastro.
![image](https://user-images.githubusercontent.com/905281/29235896-e20c9012-7ed9-11e7-9a75-21a45808768f.png)

**Comment by Nirvana16** [comment]
Jovens o seguinte : 

1) Se quiserem zoar criem varias contas a cada vez que o crédito acabar. 
2) Caso queria usar uma unica conta faça as validações, para validar o email voce ENVIA UM EMAIL pro suporte@totalvoice.com.br com seu login , telefone e informando que tentou validar e nao conseguiu. No meu caso havia um bug no site que sempre que eu clicava no link no meu email ia para uma pagina de 404. 

3) para sair do TRIAL tu tem de mandar documentação para o pessoal da Total Voice (nada de mais ) Porém nao é necessario , pode recarregar e zuar
